pandas/formats/style.py

```diff
@@ -410,6 +370,11 @@ def render(self):
         -------
         rendered: str
             the rendered HTML
+        **kwargs:
```

Comment:
you need to make the docstring a raw string (or escape the *'s), otherwise sphinx will complain :-)